ddp-workspace/projects/sandbox-app/src/app/sandboxes/activityForm/addressEmbeddedSandbox.component.ts

```diff
@@ -13,11 +13,19 @@ export class AddressEmbeddedSandboxComponent implements OnInit, OnDestroy {
   @ViewChild(AddressEmbeddedComponent, { static: true }) addressComponent: AddressEmbeddedComponent;
   public activityInstanceGuid: string;
   public inputParameters = {};
+  public isReadOnly = true;
+  public bogusAddress = null;
   private anchor: CompositeDisposable;
+  public block: MailAddressBlock;
+
```

Comment:
```suggestion
```

Reply:
And the change to public-api.ts: d010493fbee3c28c50c8a81824e275512b11049c